Source/MDKServices/HealthAnalysis.cs

```diff
@@ -76,7 +76,7 @@ static HealthAnalysis Analyze(Project project, HealthAnalysisOptions options)
                 if (!File.Exists(vrageRef))
                 {
                     options.Echo?.Invoke($"{project.Name}: Invalid game path.");
-                    analysis._problems.Add(new HealthProblem(HealthCode.BadGamePath, HealthSeverity.Critical, "Invalid game path"));
+                    analysis._problems.Add(new HealthProblem(HealthCode.BadGamePath, HealthSeverity.Warning, "Invalid game path, check Manual Path in Options"));
```

Comment:
This is a bit problematic. The system cannot work without the game path, hence why this is a critical error. What is your rationale for reducing its severity?

Reply:
If you somehow enter into a situation where you have no set gamepath, the dialog continuously loops showing you the same health analysis warning endlessly due to IsHealthy being false, this combined with the IsHealthy change means you can exit out of that loop and then proceed to check the manual path.

I couldn't see a simple way to allow the to exit out of the loop and correct the mistake since it otherwise leaves the project Unloaded and you are unable to bring up the MDK Options dialog.

It does seem however that this is an extremely strange state to get into normally. Going via the wizard sets the gamepath correctly, or at least doesn't let you continue without one, and deleting the mdk path props wouldn't be the smartest move.

Reply:
Hm. I see. 🤔 I'm gonna have to try to come up with something then. It just feels wrong to mark something as critical as this as... well, noncritical.